### ISSUE TITLE
Most avatar images not loading

### ISSUE THREAD

**Issue by improvethings** [monlovesmango/astral#57]
![Screenshot from 2022-11-02 16-36-43](https://user-images.githubusercontent.com/16601027/199622069-a0b20fdf-fa35-461c-b448-85e8cb9fd10a.png)

This might be due to third party privacy blocking within Brave browser, I'm not sure.

**Comment by monlovesmango** [comment]
yeah this is a known issue. will either need to use image proxy or find another db solution. see this post: 

http://astral.ninja/event/4b15c0d403b253577a09775be6bc166d42c35e10d3bc20ed72c110422d5bdff4

will keep you posted.

**Comment by improvethings** [comment]
Just an FYI, your reply used localhost: instead of domainname:

**Comment by monlovesmango** [comment]
doh thank you

**Comment by cryptoquick** [comment]
Just noticed this also. Looks like Damus Web has solved this issue:
https://damus.io/web/?pk=npub17jjxdns4cx3v7zdy7awj0gqsawqtezh50m7aggpvvryah8m6pgtqfs0r7j

**Comment by monlovesmango** [comment]
yeah this is not an issue on damus because damus doesn't need cors. astral has strict cors as a dependency for the db.

**Comment by cryptoquick** [comment]
Why does the database require CORS?

**Comment by monlovesmango** [comment]
it needs the shared array buffer (its an experimental db)

https://github.com/jlongster/absurd-sql#requirements

**Comment by cryptoquick** [comment]
Oh, that's neat! I know CORS can make exceptions based on resource types...
https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image

It also says I need to configure my server to support it, I think.

**Comment by monlovesmango** [comment]
yep exactly. this is already done (and why any images at all are working haha).

any image (or resource in general) that isn't served with header ```'Access-Control-Allow-Origin': *``` will err in astral unfortunately.

at some point I am going to stand up a proxy to fetch the images that run into cors issues.

**Comment by hewigovens** [comment]
Another workaround: install [CORS Unblock](https://chrome.google.com/webstore/detail/cors-unblock/lfhmikememgdcahcdlaciloancbhjino) from Chrome Web Store and only enables on astral.ninja

**Comment by monlovesmango** [comment]
ok I ended up removing the db that required astral to be cors isolated. now there shouldn't be any cors issues nor broken images. closing.
